### DISCUSSION TITLE
[Feature Request]: 豆瓣想看单独提供资源自动下载的优先级设置

### DISCUSSION THREAD

**Discussion by ThierryCheng** [jxxghp/MoviePilot#3379] [讨论]
### 当前程序版本

v1.4.0

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

豆瓣想看是一个非常有用的插件，但是当前最大的问题是虽然MP已经设置好了搜索和订阅的优先级，但是好像搜索和订阅的优先级对于豆瓣想看都无法生效。希望能单独对于豆瓣想看有一个片源优先级的定制规则，或者当前搜索的优先级对于豆瓣想看生效
比如下图我搜索和订阅的都是4K分辨率的，但是豆瓣想看添加了之后仍然会自动下载1080P的资源

<img width="1089" alt="image" src="https://github.com/jxxghp/MoviePilot/assets/8851199/455f82f8-9e6e-4686-ba6b-823911b7ae67">
<img width="1437" alt="image" src="https://github.com/jxxghp/MoviePilot/assets/8851199/a40ebbe6-8553-40c9-abf3-432451982ef7">
<img width="890" alt="image" src="https://github.com/jxxghp/MoviePilot/assets/8851199/baa7a400-494f-495d-b3b3-f5ea35211e27">

**Comment by zww94** [comment]
个人理解：优先级1 排除720p ，是不是就是可以是1080的？ 然后1080 就排在了优先级1？4k 排在了优先级2去了

**Comment by zww94** [comment]
不过能单独给豆瓣想看 设置优先级 更好
